### PR TITLE
fix(edit profile): bugs and map not showing

### LINKED ISSUE #765: Fix bug that won't allow a user to see the location map when creating a profile
- [x] Lifebanks
- [x] Sponsors

### PR DESCRIPTION
**Pull request by Edgar097**
Resolves #765 

### What does this PR do?
Fix console errors and map not showing in sponsor and Lifebank

### How should this be tested?
- Make run
- Edit profile

### CODE REVIEW

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]
Take a look at margins

image

File: hapi/src/api/user.api.js [hunk 1]
```diff
@@ -87,22 +88,27 @@ const insert = user => {
 }
 
 const setEmail = (where, email) => {
+  console.log(where, user)
```
Comment (leisterfrancisco) [inline hapi/src/api/user.api.js:91]: Check this

File: hapi/src/api/user.api.js [hunk 2]
```diff
@@ -87,22 +88,27 @@ const insert = user => {
...
   return hasuraUtils.request(SET_EMAIL, { where, email })
 }
 
 const setToken = (where, token) => {
+  console.log(where, user)
```
Comment (leisterfrancisco) [inline hapi/src/api/user.api.js:96]: Check this

File: hapi/src/api/user.api.js [hunk 3]
```diff
@@ -87,22 +88,27 @@ const insert = user => {
...
   return hasuraUtils.request(SET_TOKEN, { where, token })
 }
 
 const setName = (where, name) => {
+  console.log(where, user)
```
Comment (leisterfrancisco) [inline hapi/src/api/user.api.js:101]: Check this

File: hapi/src/api/user.api.js [hunk 4]
```diff
@@ -87,22 +88,27 @@ const insert = user => {
...
   return hasuraUtils.request(SET_NAME, { where, name })
 }
 
 const setSecret = (where, secret) => {
+  console.log(where, user)
```
Comment (leisterfrancisco) [inline hapi/src/api/user.api.js:106]: Check this

File: hapi/src/api/user.api.js [hunk 5]
```diff
@@ -87,22 +88,27 @@ const insert = user => {
...
   return hasuraUtils.request(SET_SECRET, { where, secret })
 }
 
 const verifyEmail = (where) => {
+  console.log(where)
```
Comment (leisterfrancisco) [inline hapi/src/api/user.api.js:111]: Check this

File: webapp/src/routes/EditProfile/EditProfileBank.js [hunk 6]
```diff
@@ -153,6 +152,7 @@ const EditProfileBank = ({ profile, isCompleting, onSubmit, setField, loading, u
         const userToSubmit = { ...user }
         userToSubmit.telephones = JSON.stringify(userToSubmit.telephones)
         userToSubmit.photos = JSON.stringify(user.photos)
+        console.log(userToSubmit)
```
Comment (leisterfrancisco) [inline webapp/src/routes/EditProfile/EditProfileBank.js:155]: Check this

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: hapi/src/api/user.api.js [hunk 7]
```diff
@@ -103,6 +104,7 @@ const setSecret = (where, secret) => {
...
```
Comment (leisterfrancisco) [inline hapi/src/api/user.api.js:107]: Remove console log

File: webapp/src/routes/EditProfile/EditProfileSponsorMobile.js [hunk 8]
```diff
@@ -538,16 +534,21 @@ const EditProfileSponsorMobile = ({ profile, isCompleting, onSubmit, loading })
...
-                    >
-                      <AddIcon />
-                    </IconButton>
-                  </InputAdornment>
+                  <>
+                    <InputAdornment position="end">
+                      <IconButton
+                        disabled={disablePhotoUrlInput}
+                        color="secondary"
+                        aria-label="add photo url"
```
Comment (leisterfrancisco) [inline webapp/src/routes/EditProfile/EditProfileSponsorMobile.js:542]: What if we add this to translation

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

**Review by leisterfrancisco — APPROVED** [review]

**Review by steph222 — APPROVED** [review]

**Pull request merged**